conf/layer.conf

```diff
@@ -54,3 +54,6 @@ DISTRO_FEATURES_append = " compositor thunder"
 
 # Compositor does not support vc4graphics yet
 DISTRO_FEATURES_remove = "${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', 'compositor', '', d)}"
+
+KERNEL_MODULE_AUTOLOAD += "${@bb.utils.contains("MACHINE_FEATURES", "bluetooth", "ecb", "", d)}"
+DISTRO_FEATURES_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'bluetooth', 'bluetoothcontrol', '', d)}"
```

Comment:
Can we just keep switching on bluetooth as a machine feature of adding additional distro features?